gateway-sp-comms/Cargo.toml

```diff
@@ -27,7 +30,7 @@ thiserror.workspace = true
 tlvc.workspace = true
 tokio.workspace = true
 usdt.workspace = true
-uuid.workspace = true
+uuid = { workspace = true, features = ["v4"] }
```

Comment:
This was necessary for unrelated reasons: the task dump code added in #316 uses `Uuid::new_v4` in `gateway-sp-comms`, but `gateway-sp-comms` doesn't enable the `"v4"` feature (only `faux-mgs` does). So, this didn't compile for me using the v2 cargo feature resolver.